backend/package.json

```diff
@@ -8,6 +8,7 @@
   "main": "src/app.js",
   "scripts": {
     "ci": "npm run lint && npm test",
+    "kinesis": "node bin/kinesis.js",
```

Comment:
Actually now that I think about it, writing docs for all the different things you need to start is going to be a pain, and then it's all going to get deleted as soon the other PR (docker stuff) happens. So I'm inclined to leave this for now, on the condition that dockerising the other bits a pieces happens straight away.